mapswipe_workers/tests/fixtures.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+
+from tests.integration.set_up import upload_file_to_firebase
+
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+FIXTURE_DIR = os.path.join(TEST_DIR, "fixtures")
+
+
+def get_fixture(path):
+    file_path = os.path.join(FIXTURE_DIR, path)
+    with open(file_path) as file:
+        return json.load(file)
+
+
+def create_test_draft(
+    fixture_path: str,
+    fixture_name: str = "user",
+    identifier: str = "",
```

Comment:
How about always generating a random identifier in this function? E.g. using `UUID` module